search/internals.go

```diff
@@ -94,17 +94,17 @@ func setupFuncs(g graph.Graph, cost graph.CostFunc, heuristicCost graph.Heuristi
 
 /** Sorts a list of edges by weight, agnostic to repeated edges as well as direction **/
 
-type edgeSorter []concrete.WeightedEdge
+type byWeight []concrete.WeightedEdge
 
-func (e edgeSorter) Len() int {
+func (e byWeight) Len() int {
```

Comment:
I don't think there is a strong convention on this; have a look at the examples in sort. The two choices I would make are from {'e', 'w'}. I don't think it matters that much.